software/open-slides.yml

```
@@ -0,0 +1,13 @@
+name: "OpenSlides"
+website_url: "https://openslides.com/"
+source_code_url: "https://github.com/OpenSlides/OpenSlides"
+description: "A web based presentation and assembly system for managing and projecting agenda, motions and elections of an assembly."
+licenses:
+  - MIT
+platforms:
+  - Docker
+tags:
+  - Conference Management
+  - Booking and Scheduling
+  - Polls and Events
```

```suggestion
tags:
  - Conference Management
  - Booking and Scheduling
```

`Polls and Events` is specifically for online polls